### ISSUE TITLE
RMG simulation stuck

### ISSUE THREAD

**Issue by Jakubanoo** [ReactionMechanismGenerator/RMG-Py#2150]
Hey everyone!

I'm trying to run a diethyl sulfide mechanism simulation.

The reaction stops after 'For reaction generation 1 process is used.', and does not move any further.

I attached input file, as well as last relevant part of RMG.log.

[input.txt](https://github.com/ReactionMechanismGenerator/RMG-Py/files/6602422/input.txt)
[RMG.txt](https://github.com/ReactionMechanismGenerator/RMG-Py/files/6602432/RMG.txt)

Im running the simulation on ubuntu, source with anaconda, and i am using latest rmg version.

Please help me understand where am i screwing up.

Thank you!

**Comment by mjohnson541** [comment]
To speed things up you can very safely add filterReactions=True to the model block and add a species constraints block: generatedSpeciesConstraints(
    maximumRadicalElectrons=2,
)

If you don't think birads are important you can change that to 1, you can also add constraints for particular numbers of atoms. However the unfortunate reality is that for large species in RMG it can take a while to react and the above modifications aren't going to stop RMG from hanging when reacting. It's a bit unusual to see it happen this early in a run, but you're seeing some molecular growth (not sure if that's real or related to errors in the thermochemistry). My recommendation is to either add additional species constraints to the above or give it a day see if it makes more progress.

**Comment by Jakubanoo** [comment]
Firstly, thank you very much for giving me an answer.

I'll try to run it with maximumRadicalElectrons=2 and give it some time. If this won't help, i'll constrain more species and give it a go.

Once more, thank you.

**Comment by Jakubanoo** [comment]
> To speed things up you can very safely add filterReactions=True to the model block and add a species constraints block: generatedSpeciesConstraints(
> maximumRadicalElectrons=2,
> )
> 
> If you don't think birads are important you can change that to 1, you can also add constraints for particular numbers of atoms. However the unfortunate reality is that for large species in RMG it can take a while to react and the above modifications aren't going to stop RMG from hanging when reacting. It's a bit unusual to see it happen this early in a run, but you're seeing some molecular growth (not sure if that's real or related to errors in the thermochemistry). My recommendation is to either add additional species constraints to the above or give it a day see if it makes more progress.


Finally, after some time i got an error (input and whole log attached)
[input.txt](https://github.com/ReactionMechanismGenerator/RMG-Py/files/6603713/input.txt)
[RMG.txt](https://github.com/ReactionMechanismGenerator/RMG-Py/files/6603714/RMG.txt)

"At time 1.4311e-02 s, reached target termination conversion: 0.900000 of DES(1)
    DES(1) conversion: 0.901     

For reaction generation 1 process is used.
Generating thermo for new species...
Traceback (most recent call last):
  File "rmg.py", line 118, in <module>
    main()
  File "rmg.py", line 112, in main
    rmg.execute(**kwargs)
  File "/home/jakub/RMG/RMG-Py/rmgpy/rmg/main.py", line 923, in execute
    trimolecular_react=self.trimolecular_react)
  File "/home/jakub/RMG/RMG-Py/rmgpy/rmg/model.py", line 614, in enlarge
    self.apply_thermo_to_species(procnum)
  File "/home/jakub/RMG/RMG-Py/rmgpy/rmg/model.py", line 824, in apply_thermo_to_species
    self.generate_thermo(spc, rename=True)
  File "/home/jakub/RMG/RMG-Py/rmgpy/rmg/model.py", line 831, in generate_thermo
    submit(spc, self.solvent_name)
  File "/home/jakub/RMG/RMG-Py/rmgpy/thermo/thermoengine.py", line 175, in submit
    spc.thermo = evaluator(spc, solvent_name=solvent_name)
  File "/home/jakub/RMG/RMG-Py/rmgpy/thermo/thermoengine.py", line 160, in evaluator
    thermo = generate_thermo_data(spc, solvent_name=solvent_name)
  File "/home/jakub/RMG/RMG-Py/rmgpy/thermo/thermoengine.py", line 125, in generate_thermo_data
    thermo0 = thermodb.get_thermo_data(spc)
  File "/home/jakub/RMG/RMG-Py/rmgpy/data/thermo.py", line 1406, in get_thermo_data
    thermo0 = self.get_thermo_data_from_groups(species)
  File "/home/jakub/RMG/RMG-Py/rmgpy/data/thermo.py", line 1876, in get_thermo_data_from_groups
    tdata = self.estimate_thermo_via_group_additivity(molecule)
  File "/home/jakub/RMG/RMG-Py/rmgpy/data/thermo.py", line 2112, in estimate_thermo_via_group_additivity
    thermo_data = self.estimate_radical_thermo_via_hbi(molecule, self.compute_group_additivity_thermo)
  File "/home/jakub/RMG/RMG-Py/rmgpy/data/thermo.py", line 2023, in estimate_radical_thermo_via_hbi
    thermo_data_sat = stable_thermo_estimator(saturated_struct)
  File "/home/jakub/RMG/RMG-Py/rmgpy/data/thermo.py", line 2150, in compute_group_additivity_thermo
    data_added = self._add_group_thermo_data(thermo_data, self.groups['group'], molecule, {'*': atom})[1]
  File "/home/jakub/RMG/RMG-Py/rmgpy/data/thermo.py", line 2543, in _add_group_thermo_data
    raise DatabaseError(f"Node {node.label} points to a non-existing group called {data} "
rmgpy.exceptions.DatabaseError: Node S6ddd points to a non-existing group called S6ddd-XdXdXd in database group"

Not sure what to do with this.

Thank you in advance.

**Comment by sthakris** [comment]
I exactly have same issue when doing for large molecules TNT and PETN. I tried different things over 1 week period and nothing worked. I went through documentation, git hub similar issues and try to find solution by different hints given at different issues. Still after more than 10 hours and so it is does not move forward and gets stuck there forever. 
I hope this issues can be fixed by solver tweaking or anything else and help would be appreciated.
